html-template/index.html

```diff
@@ -17,7 +17,10 @@
   <link rel='stylesheet' href='<%= htmlWebpackPlugin.options.publicUrl %>/styles/grid-old-ie.css' type='text/css'>
   <![endif]-->
 
-   <link href="/static/stylesheets/application.css" rel="stylesheet" />
+  <link href="/static/stylesheets/application.css" rel="stylesheet" />
+
+  <link rel='stylesheet' href='<%= htmlWebpackPlugin.options.frontendComponentPath %>/assets/stylesheets/header.css' type='text/css' nonce='<%= htmlWebpackPlugin.options.nonce %>'>
+  <link rel='stylesheet' href='<%= htmlWebpackPlugin.options.frontendComponentPath %>/assets/stylesheets/footer.css' type='text/css' nonce='<%= htmlWebpackPlugin.options.nonce %>'>
```

Comment:
In the Nunjucks template we include the css with the response from the endpoint with `cssIncludes: [...header.css, ...footer.css]` however here I'm hardcoding the path to each css because I was having difficulties passing it at runtime

Other suggestions welcome!